.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @spanglerco @cameronwaterman @maxgax @prestwick @chris468
+* @JD-Robertson @cameronwaterman @maxgax @prestwick @chris468
```

Comment:
Looks like JD isn't in the NI org yet, so this won't actually add him as an owner. Also, feel free to suggest other squad members to be added to this list if desired @maxgax.